### ISSUE TITLE
hdr not being generated

### ISSUE THREAD

**Issue by DaniloMussolini** [ISDCF/Sign-Language-Video-Encoding#2]
I'm not sure if this is related to the media files I'm using, but the chunk.hdr file is not being generated in the temp folder. I have tried the script in both systems, Linux e MacOS.

Bellow is an output example:

`[root@localhost vp9]# perl encode_vp9_wav.perl cilla_black_bean_sauce.mp4 
Looking to see if resizing is necessary...
Encoding with:
  Frame Rate: 25 fps
  Chunk Length: 50 frames
  Max Chunk Size: 288000 bytes
  Bitrate: 576000
  Resizing: -filter:v scale=w=480:h=640:force_original_aspect_ratio=decrease
Success! Encoded all chunks to vp9
read_file '/tmp/Q4AM4wiSLz/chunk.hdr' - sysopen: No such file or directory at encode_vp9_wav.perl line 105.
`

**Comment by lucquer4** [comment]
I encountered the same issue on both my Ubuntu and Debian installations running within the Windows 11 WSL environment. Rather than opting for the official version available through APT repositories, I chose to compile FFMpeg 3.2.4 from its source, as recommended in the README.md file.

In addition, I found it necessary to install the "libfile-slurp-perl" package. However, following this installation, I encountered an error regarding the absence of 'chunk.hdr'.

Subsequently, I opted to start afresh by uninstalling FFMpeg and then reinstalling it from the official repository, thereby acquiring version 4.4.2. Upon rerunning the command, I achieved success.

In summary, the steps are as follows:

On a freshly installed Debian or Ubuntu system within Windows:

- Install FFMpeg: sudo apt install ffmpeg (the default version available from the repository is 4.4.2 on Ubuntu 22.04 LTS)
- Install required Slurp File: sudo apt install libfile-slurp-perl
- Run the script: ./encode-vp9-wav [Your VP9 Video File].webm
- SUCCESS

**Comment by lucquer4** [comment]
I've tested the Sign Language decoding with the "decode-vp9-wav" script and with easyDCP Player 4.0.5
